CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - Issue 18 - Remove Flask
     - Issue 52 - Remove partial_f from data columns to obscure
 ### Fixed
+    - Issue 44 - Load data lambda only loads the first granule found in the time range
```

Comment:
Can you move this to the unreleased section please. All changes being merged to `develop` should be described in the Unreleased section of the changelog. Part of creating a release is renaming the Unreleased section to the new release and starting a new blank Unreleased section for tracking the next set of changes.